grpc/proto/disruption_listener.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package disruption_listener;
+
+option go_package = "github.com/DataDog/dd-go/domains/chaos-tooling/apps/chaos-grpc-dogfood/chaos-interceptors/disruption_listener";
```

Comment:
I think it'd be best to bring all the code we need into this repo, rather than leak this directory structure